contracts/Components.cdc

```diff
@@ -259,17 +259,17 @@ pub contract Components {
         }
     }
 
-    pub resource Admin {
-        pub let options: {String: {String: Bool}}
-        pub let colors: {String: Bool}
+    access(all) resource Admin {
+        access(all) let options: {String: {String: Bool}}
+        access(all) let colors: {String: Bool}
 
-        pub fun createRandom(): Renderer {
+        access(all) fun createRandom(): Renderer {
             let c: [String] = []
             var count = 0
             while count < 7 {
                 count = count + 1
-
-                c.append(self.colors.keys[unsafeRandom() % UInt64(self.colors.keys.length)])
+                
+                c.append(self.colors.keys[revertibleRandom<UInt64>(modulo: UInt64(self.colors.keys.length))])
```

Comment:
neat! Not familiar with this part of syntax changes. Looks like they've made a build-in way to get things in a desired range?